R/util.R

```diff
@@ -3,8 +3,9 @@
 #' @param board_params board_params (only the alias needed)
 #' @keywords internal
 dpconnect_check <- function(board_params) {
```

Comment:
Probably should just remove this function -- with new refactoring for pins v1 this is no longer relevant